### PR TITLE
[86by8d89p][breadcrumbs] some a11y improvements

### PR DESCRIPTION
**Pull request by ilyabrower**
## Motivation and Context
Improve a11y for breadcrumbs

## How has this been tested?
Manually

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [X] Nice improve.

## Checklist:

- [X] My code follows the code style of this project.
- [X] I have updated the documentation accordingly or it's not required.
- [X] Unit tests are not broken.
- [X] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [ ] I have added new unit tests on added of fixed functionality.

### CODE REVIEW

File: semcore/breadcrumbs/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [5.29.1] - 2024-04-30
+
+### Fixed
+
+- Some a11y improvements.
```
Comment (msereniti) [inline semcore/breadcrumbs/CHANGELOG.md:9]: We should provide more detailed changelog

File: semcore/breadcrumbs/src/style/breadcrumbs.shadow.css [hunk 2]
```diff
@@ -3,46 +3,59 @@ SBreadcrumbs {
...
   padding: 0;
   margin: 0;
-  list-style: none;
-}
 
-SSeparator {
-  margin: 0 var(--intergalactic-spacing-2x, 8px);
-  color: var(--intergalactic-icon-secondary-neutral, #a9abb6);
-  display: flex;
+  SOl {
```
Comment (msereniti) [inline semcore/breadcrumbs/src/style/breadcrumbs.shadow.css:7]: Probably more semantic name is needed here

File: semcore/breadcrumbs/src/style/breadcrumbs.shadow.css [hunk 3]
```diff
@@ -3,46 +3,59 @@ SBreadcrumbs {
...
+    display: flex;
+    align-items: center;
+    list-style: none;
+    overflow: hidden;
+    margin: 0;
+    padding: 0;
```
[the remark this answers is not in the document]
Comment (ilyabrower) [reply semcore/breadcrumbs/src/style/breadcrumbs.shadow.css:13]: yes, changed

File: semcore/breadcrumbs/src/style/breadcrumbs.shadow.css (hunk 2 above)
Comment (ilyabrower) [reply semcore/breadcrumbs/src/style/breadcrumbs.shadow.css:7]: changed

File: semcore/breadcrumbs/CHANGELOG.md (hunk 1 above)
Comment (ilyabrower) [reply semcore/breadcrumbs/CHANGELOG.md]: changed

**Pull request closed without merging**